OpenRA.Game/Network/GameServer.cs

```diff
@@ -33,6 +34,7 @@ public class GameClient
 		public GameClient(Session.Client c)
 		{
 			Name = c.Name;
+			Fingerprint = c.Fingerprint;
```

Comment:
Just curious, where is this used? I might have just missed it, but it doesn't look like any of this PR's other changes use this.

Reply:
It's not used yet, but at some point in the future we may want to expose registered accounts in the game or web server browsers.